investing.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, render_template, request
+
+#Written by ChatGPT #AIGO #AIBEST #AIGO #GOFUND #ML #DEEPLEARNING #COMPUTER_VISION #AGI #NO_BIOLOGICAL_INTELLIGENCE
+
+app = Flask(__name__)
+@app.route('/credit-card-info', methods=['GET', 'POST'])
```

Comment:
Why do you have both `POST` and `GET` requests here ?
you can get rid of the `GET` request (redundant and never used)